packages/components/inputs/radio-input/src/radio-option.styles.ts

```diff
@@ -1,36 +1,33 @@
+// TODO: @redesign cleanup
 import { css } from '@emotion/react';
 import styled from '@emotion/styled';
 import { designTokens } from '@commercetools-uikit/design-system';
-import { TOptionProps } from './radio-option';
+import type { TStylesProps } from './radio-option';
 
-type TStylesProps = Pick<
-  TOptionProps,
-  'isDisabled' | 'hasError' | 'hasWarning' | 'isHovered' | 'isReadOnly'
->;
+const getDefaultThemeLabelColor = (props: TStylesProps) => css`
+  color: ${props.isDisabled
+    ? designTokens.fontColorForInputWhenDisabled
+    : designTokens.fontColorForInput};
+`;
 
 const LabelTextWrapper = styled.div<TStylesProps>`
   grid-area: label;
-  margin-left: ${designTokens.spacing20};
+  margin-left: ${designTokens.marginLeftForRadioInputLabel};
   font-size: 1rem;
   font-family: inherit;
-  color: ${(props) =>
-    props.isDisabled
-      ? designTokens.fontColorForInputWhenDisabled
-      : designTokens.fontColorForInput};
+  display: flex;
+  ${(props) => !props.isNewTheme && getDefaultThemeLabelColor(props)}
```

Comment:
Nit: you can handle the if-else logic within the `getDefaultThemeLabelColor` function.

This then becomes simply:

```suggestion
  ${getDefaultThemeLabelColor}
```